tests/lib/cla_check.py

```diff
@@ -73,7 +73,7 @@ def static_email_check(email, master_emails, width):
                 yellow, reset, email, width
             )
         )
-        return True
+        return False
```

Comment:
There's a couple of @users.noreply.github.com email addresses in master already which will be caught by the email in master_emails branch above. The addresses I was able to find belong to Canonical guys anyway:
- chipaca@users.noreply.github.com
- gerboland@users.noreply.github.com